edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -65,17 +66,17 @@ def jwt_decode_handler(token):
         InvalidTokenError: Decoding fails.
     """
     jwt_issuer = get_first_jwt_issuer()
-    _verify_jwt_signature(token, jwt_issuer)
+    _verify_jwt_signature(token, jwt_issuer, decode_symmetric_token=decode_symmetric_token)
     decoded_token = _decode_and_verify_token(token, jwt_issuer)
     return _set_token_defaults(decoded_token)
 
 
-def configured_jwt_decode_handler(token):
+def configured_jwt_decode_handler(token, decode_symmetric_token=True):
     """
     Calls the ``jwt_decode_handler`` configured in the ``JWT_DECODE_HANDLER`` setting.
     """
     api_setting_jwt_decode_handler = api_settings.JWT_DECODE_HANDLER
-    return api_setting_jwt_decode_handler(token)
+    return api_setting_jwt_decode_handler(token, decode_symmetric_token)
```

Comment:
Hmm. I don't think we can just change the signature of this setting that expects a handler that just takes a token. However, I (or you) will need to think about what this means.

For example, here is the ecommerce "configured" handler: https://github.com/GetBlimp/django-rest-framework-jwt

We may need a separate method that doesn't get the configured handler, but just uses the drf-extensions handler to do some post-processing? These are just thoughts off the top of my head so we can think further about this.